### ISSUE TITLE
Cannot set alignBottom using the setOptions method

### ISSUE THREAD

**Issue by GoogleCodeExporter** [seancallinan/google-maps-utility-library-v3#77]
```
Demo link:
http://www.giveblood.org/locations-test2 

Code:
http://giveblood.org/uploads/js/map.locations3.js

What steps will reproduce the problem?

I am trying to set the alignBottom to true using the setOptions method, but it 
has no effect. I am able to set the pixelOffset and content within the same 
method without issue (see line 177 of js code). The alignBottom works if I 
declare it within my options in the constructor (starting at line 50 in js 
code), but I need to change the alignBottom option on certain markers.

I am trying to alter the alignBottom based on the marker location, if the 
marker is in the southern half the alignBottom would be true, if in the 
northern half, false. 

Version: 1.1.5

Browser / Operating System:
Chrome,FF,IE / Windows XP


*********************************************************
Tip: Star this issue (next to title) to receive notifications of status
changes against this issue, also used as a gauge for how many people are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `gare...@gmail.com` on 6 Apr 2011 at 2:08

**Comment by GoogleCodeExporter** [comment]
```
Forgot to mention this in the InfoBox library.
```

Original comment by `gare...@gmail.com` on 6 Apr 2011 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in the current trunk version of InfoBox (1.1.7). Please 
verify.

See: 
http://google-maps-utility-library-v3.googlecode.com/svn/trunk/infobox/src/infob
ox.js
```

Original comment by `garylitt...@gmail.com` on 8 Jul 2011 at 4:56
- Changed state: **FixedNotReleased**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the 1.1.7 tagged release.
```

Original comment by `garylitt...@gmail.com` on 11 Jul 2011 at 4:36
- Changed state: **Fixed**
